webapp/tests/karma/ts/modules/messages/messages-content.component.spec.ts

```diff
@@ -245,5 +246,51 @@ describe('MessagesContentComponent', () => {
       expect(removeMessageFromSelectedConversationSpy.args[0]).to.deep.equal(['message2']);
     });
   });
+
+  describe.only('New message section', () => {
+    it('should display the "new message" section in a conversation with a known recipient', async () => {
+      component.selectedConversation = {
+        'id': '7aeaa4bc-ae5e-4964-8771-697209d2c6dd',
```

Comment:
Quotes around property titles are unnecessary and just clutter the view. Please only use them when they're needed: fe for composite property titles like 'this-is-a-property-title'.

Reply:
You're right! I would expect the linter to pick it up, should we add a rule for this?

Reply:
Good call - it should be configurable with the quote-props option: https://eslint.org/docs/rules/quote-props

Reply:
I've added the rule and fixed lint errors it created in the following files:
```
check-date.service.spec.ts
constants.spec.ts
constants.ts
contacts.spec.ts
parse.provider.spec.ts
privacy-policies.service.spec.ts
telemetry.service.spec.ts
translate-from.service.spec.ts
translate-message-formate-compiler.provider.spec.ts
uhc-stats.service.spec.ts
update-user.service.ts
user-login.service.ts
xml-forms.service.spec.ts
```